### PR TITLE
[TOOLS-251] Improve authentication failure handling. Rotate "updating…

### PR DESCRIPTION
**Pull request by taksan**
…" icon

### CODE REVIEW

File: src/main/java/objective/taskboard/config/SecurityConfiguration.java [hunk 1]
```diff
@@ -76,7 +75,11 @@ public Authentication authenticate(Authentication authentication) throws Authent
                 String name = authentication.getName();
                 String password = authentication.getCredentials().toString();
 
-                jiraService.authenticate(name, password);
+                try {
+                    jiraService.authenticate(name, password);
+                }catch(Exception e) {
```
Comment (objective-solutions-user) [inline src/main/java/objective/taskboard/config/SecurityConfiguration.java]: MAJOR Either log or rethrow this exception. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1166)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/main/resources/static/elements/taskboard/taskboard-home.html [hunk 2]
```diff
@@ -50,19 +50,37 @@
...
+            -webkit-animation-iteration-count: infinite;
+            -webkit-animation-timing-function: linear;
+            -moz-animation-name: spin;
+            -moz-animation-duration: 4000ms;
+            -moz-animation-iteration-count: infinite;
+            -moz-animation-timing-function: linear;
+            -ms-animation-name: spin;
+            -ms-animation-duration: 4000ms;
+            -ms-animation-iteration-count: infinite;
+            -ms-animation-timing-function: linear;
```
Comment (jhonypg) [inline src/main/resources/static/elements/taskboard/taskboard-home.html]: Default "animation" property is missing.

File: src/main/resources/static/elements/taskboard/taskboard-home.html [hunk 3]
```diff
@@ -50,19 +50,37 @@
...
         }        
 
         .issue-state-initializing {
             color: #3da83d;
+            
+            -webkit-animation-name: spin;
+            -webkit-animation-duration: 4000ms;
+            -webkit-animation-iteration-count: infinite;
+            -webkit-animation-timing-function: linear;
```
Comment (jhonypg) [inline src/main/resources/static/elements/taskboard/taskboard-home.html]: Default "animation" property is missing.

**Pull request merged**